docs/guides/modules/test/pages/smarter-testing.adoc

```diff
@@ -41,565 +41,1571 @@ TIP: In codebases with sparse test coverage, Smarter Testing cannot accurately d
+. Confirm the analysis command is producing valid coverage output, and that your are using the appropriate `outputs` variable for the coverage format.
+
+==== Test impact analysis finds many more files impacting a test than it should
+
+Analysis is based off the code covered by running an individual test atom. Sometimes the language runtime or test runner will eagerly load code as part of the test run setup. This behavior can lead to all the eagerly loaded code being considered covered by the test atom though it is only an artifact of running the test suite.
+
+In these cases you can set an <<analysis-baseline-command,analysis-baseline command>> which can account for this code coverage.
+
+
+== 3. Run your test suite in your CircleCI job
+
+Now that your test suite configuration is set up correctly you can run your test suite in CircleCI job.
+
+. Make sure that `.circleci/test-suites.yml` is checked in to your repository.
+. Update your `.circleci/config.yml` to call the `circleci run testsuite "ci tests"` command instead of your regular test command.
+. Push the change to your VCS.
+
+E.g. if your CircleCI test job was:
+[source,yaml]
+----
+version: 2.1
+jobs:
+  test:
+    executor: node-with-service
+    steps:
+      - setup
+      - run: vitest run --reporter=junit --outputFile="test-reports/tests.xml" --bail 0
+      - store_test_results:
+          path: test-reports
+----
+
+You would change it to:
+[source,yaml]
+----
+version: 2.1
+jobs:
+  test:
+    executor: node-with-service
+    steps:
+      - setup
+      - run: circleci run testsuite "ci tests"
+      - store_test_results:
+          # This directory must match the directory of `outputs.junit` in your
+          # test-suites.yml
+          path: test-reports
+----
+
+
+Since there is no test impact analysis data stored in CircleCI for your test suite, all the tests will be selected and run when you push to CI. Confirm that the tests execute as expected.
+
+TIP: It can be easier to debug issues with your test suite configuration in CI if you initially run the job without parallelism.
+
+The analysis phase runs on the default branch, you can pre-populate the test impact analysis data before merging by temporarily adding the `--test-analysis=impacted` CLI flag to the test suite invocation:
+[source,yaml]
+----
+version: 2.1
+jobs:
+  test:
+    executor: node-with-service
+    steps:
+      - setup
+      # override test selection and analysis defaults to perform analysis on a
+      # feature branch
+      - run: circleci run testsuite "ci tests" --test-selection=none --test-analysis=impacted
+      - store_test_results:
+          path: test-reports
+----
+
+Analysis will be limited to approximately 15 minutes due to `options.test-analysis-duration` in the test suite configuration. If analysis isn't completing within those 15 minutes, you can remove `options.test-analysis-duration` to allow the process to run for as long as it needs.
```

Comment:
How likely is it that a customer's first analysis will take under 15 minutes? If it's unlikely, we could just add a step to remove the 15-minute limit.
Another option is to keep the 15-minute limit and add guidance that users can speed up analysis by:
- Increasing parallelism
- Using a scheduled pipeline